stdlib/area/position.lua

```diff
@@ -201,4 +201,21 @@ function Position.tostring(pos)
     end
 end
 
+--- Returns the opposite direction - Adapted from Factorio util.lua
+-- @param direction the defines.direction to get the opposite of
+-- @return the opposite direction
+function Position.opposite_direction(direction)
+    local opposites = {
```

Comment:
Uhh, shouldn't this be hoisted outside of the method call so it is not rebuilt on every single invocation of `Position.opposite_direction`?